package.json

```diff
@@ -43,20 +43,20 @@
         "test": "vitest run"
     },
     "devDependencies": {
-        "@types/node": "^18.11.9",
+        "@types/node": "^20.19.0 || >=22.12.0",
         "@typescript-eslint/eslint-plugin": "^5.21.0",
         "@typescript-eslint/parser": "^5.21.0",
         "esbuild": "0.16.10",
         "eslint": "^8.14.0",
         "typescript": "^4.6.4",
-        "vite": "^6.0.0",
+        "vite": "^7.0.0",
         "vitest": "^0.34.4"
     },
     "peerDependencies": {
-        "vite": "^5.0.0 || ^6.0.0"
+        "vite": "^7.0.0"
```

Comment:
As the package is extremely stable, I don't feel we need to support multiple versions of Vite anymore.

We will tag a major version when we release this.